main.py

```diff
@@ -15,16 +15,15 @@ def main():
 
     # The context that will manage the airbrakes state machine
     airbrakes = AirbrakesContext(logger, servo, imu)
-
-    # Start the IMU and logger processes:
-    airbrakes.start()
-
-    # This is the main loop that will run until the stop method on the airbrakes is called
-    while not airbrakes.shutdown_requested:
-        airbrakes.update()
-
-    # Shutdown the IMU and logger processes:
-    airbrakes.stop()
+    try:
+        airbrakes.start()  # Start the IMU and logger processes
+        # This is the main loop that will run until we press Ctrl+C
+        while not airbrakes.shutdown_requested:
+            airbrakes.update()
+    except KeyboardInterrupt:
```

Comment:
I don't think that a keyboard interrupt will stop the main process when the rocket is actually being. Launched, it will probably just end when we unplug the battery. That being said, we will end sims via keyboard interrupt, so I think its good.